plugins/woocommerce/client/admin/client/order-attribution-install-banner/use-order-attribution-install-banner.js

```diff
@@ -128,10 +128,12 @@ export const useOrderAttributionInstallBanner = () => {
 				loadingRecommendations: ! hasFinishedResolution(
```

Comment:
When `canUserInstallPlugins` is `false`, `loadingRecommendations` will always stay `true` because the request to the recommendations API was never started.

This doesn't seem to cause other side effects or even issues, but it's still recommended to make it conditionally determine its state according to `canUserInstallPlugins`.